applications/GeoMechanicsApplication/tests/test_thermal_element/test_thermal_line_element/test_thermal_line_element_3D2N/test_thermal_line_element_3D2N.mdpa

```diff
@@ -0,0 +1,100 @@
+Begin Properties 1
+End Properties
+
+
+Begin Nodes
+ 1    0.000000e+00    0.000000e+00    0.000000e+00
+ 2    0.000000e+00   -1.000000e+00    0.000000e+00
+ 3    0.000000e+00   -2.000000e+00    0.000000e+00
+ 4    0.000000e+00   -3.000000e+00    0.000000e+00
+End Nodes
+
+
+Begin Elements GeoTransientThermalLineElement2D2N
+  1  1  1  2
+  2  1  2  3 
+  3  1  3  4
+End Elements 
+
+
+Begin SubModelPart Start
+  Begin SubModelPartTables
+  End SubModelPartTables
+  Begin SubModelPartNodes
+    1
+    2
+    3
+    4
+  End SubModelPartNodes
+  Begin SubModelPartElements
+    1
+    2
+    3
+  End SubModelPartElements
+  Begin SubModelPartConditions
+  End SubModelPartConditions
+End SubModelPart
+
+
+Begin SubModelPart filter
+  Begin SubModelPartTables
+  End SubModelPartTables
+  Begin SubModelPartNodes
+    1
+    2
+    3
+    4
+  End SubModelPartNodes
+  Begin SubModelPartElements
+    1
+    2
+    3
+  End SubModelPartElements
+  Begin SubModelPartConditions
+  End SubModelPartConditions
+ End SubModelPart
+
+
+Begin SubModelPart top
+  Begin SubModelPartTables
+  End SubModelPartTables
+  Begin SubModelPartNodes
+    1
+  End SubModelPartNodes
+  Begin SubModelPartElements
+  End SubModelPartElements
+  Begin SubModelPartConditions
+  End SubModelPartConditions
+End SubModelPart
+
+
+Begin SubModelPart bottom
+  Begin SubModelPartTables
+  End SubModelPartTables
+  Begin SubModelPartNodes
+    4
+  End SubModelPartNodes
+  Begin SubModelPartElements
+  End SubModelPartElements
+  Begin SubModelPartConditions
+  End SubModelPartConditions
+End SubModelPart
+
+
+Begin SubModelPart Gravity
+  Begin SubModelPartTables
+  End SubModelPartTables
+  Begin SubModelPartNodes
+    1
+    2
+    3
+    4
+  End SubModelPartNodes
+  Begin SubModelPartElements
+    1
+    2
+    3
+  End SubModelPartElements
+  Begin SubModelPartConditions
+  End SubModelPartConditions
+End SubModelPart
```

Comment:
This sub-model part doesn't appear to be used. If that is correct, please remove it.